lib/membrane_h264_plugin/parser/caps.ex

```diff
@@ -8,7 +8,7 @@ defmodule Membrane.H264.Parser.Caps do
 
   @default_caps %H264{
     alignment: :au,
-    framerate: {30, 1},
+    framerate: {0, 1},
```

Comment:
I would set the default for this the same as for the `framerate` option in Parser element

Reply:
Now it is set to `nil`, as the Parser now uses a dependency to h264_format v0.5.0